tests/unit/api/test_feature_list.py

```diff
@@ -634,8 +629,6 @@ def test_list_versions(saved_feature_list):
                 "id": [saved_feature_list.id],
                 "name": [saved_feature_list.name],
                 "version": [saved_feature_list.version],
-                "feature_list_namespace_id": [saved_feature_list.feature_list_namespace.id],
-                "num_feature": 1,
```

Comment:
For `feature_list.list_versions()`, we remove both `feature_list_namespace_id` and `num_feature`.